client/app/vps/import/vps.service.js

```diff
@@ -674,34 +674,35 @@ angular.module("managerApp").service("VpsService", [
             });
         };
 
-        // HOT FIX remove this fukin shit
-        this.getOptionDetails2 = function (option, vps, duration) {
+        // HOT FIX
+        this.getSnapshotDetails = function (vps, duration) {
+            const version = vps.version.replace(/_/g, "").toLowerCase();
```

Comment:
Could we have an example in comments please ?

`// from "_2018_V_1" to "2018v1"`